src/glang/lib/std/sys/types.c3

```diff
@@ -1,5 +1,15 @@
 // sys/types.h - data types
 // https://pubs.opengroup.org/onlinepubs/009695399/basedefs/sys/types.h.html
 
+typedef blkcnt_t : isize
+typedef blksize_t : isize
+typedef dev_t : u64
+typedef gid_t : id_t
+typedef id_t : u32
+typedef ino_t : u64
+typedef mode_t : u32
+typedef nlink_t : u64
 typedef off_t : isize
 typedef pid_t : int
+typedef time_t : isize
+typedef uid_t : id_t
```

Comment:
These are also platform-dependent and should be auto-generated.

Reply:
I've removed these for now. I'm trying to take stat directly from the kernel implementation instead.